server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/ElementItem.java

```diff
@@ -37,11 +37,12 @@ public ElementItem(
       int levelNumber,
       String name,
       Locality definition,
+      Boolean global,
```

Comment:
why not primitive boolean?

Reply:
I was adding and deleting them a couple of times while implementing it. I have used the null ability in my previous versions. But not now and I need to rewrite it.